docs/section6/Rule6-13.md

```diff
@@ -0,0 +1,78 @@
+
+# Lighting - Rule 6-13
+
+**Rule ID:** 6-13  
+**Rule Description:** Additional occupancy sensor controls in the proposed building are modeled through schedule adjustments based on factors defined in Table G3.7.  
+**Rule Assertion:** Proposed RMR = expected value  
+**Appendix G Section:** Section G3.1-6(i) Modeling Requirements for the Proposed design  
+**Appendix G Section Reference:**  
+
+- Table G3.7, Performance Rating Method Lighting Power Density Allowances and Occupancy Sensor Reductions Using the Space-by-Space Method  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:**  
+
+  1. Pass Rule 6-10.
+
+**Manual Check:** Yes  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** Table G3.7  
+**Function Call:**  
+
+  - compare_schedules()
+  - normalize_space_lighting_schedule()
+
+## Rule Logic:  
+
+- **Applicability Check 1:** If pass Rule 6-10: `if rule_6_10 == TRUE:`  
+
+- Get building open schedule in the proposed model: `building_open_schedule_p = P_RMR.building.building_open_schedule`  
+
+- For each building segment in building: `for building_segment_p in P_RMR.building.building_segments:`  
+
+  - For each thermal block in building segment: `thermal_block_p in building_segment_p.thermal_blocks:`  
+
+    - For each zone in thermal block: `zone_p in thermal_block_p.zones:`  
+
+      - For each space in zone: `space_p in zone_p.spaces:`  
+
+        - Get matching space from B_RMR: `space_b = match_data_element(B_RMR, Spaces, space_p.id)`  
+
+          - Get normalized space lighting schedule for B_RMR: `normalized_schedule_b = normalize_space_lighting_schedule(space_b)`  
+
+        - Get lighting space type: `lighting_space_type_p = space_p.lighting_space_type`  
+
+          - If lighting space type is employee lunch and break rooms, conference/meeting rooms, or classrooms (not including shop classrooms, laboratory classrooms, and preschool through 12th-grade classrooms), get baseline lighting occupancy sensor reduction factor from Table G3.7: `if lighting_space_type_p in ["LOUNGE/BREAKROOM", "CONFERENCE/MEETING/MULTIPURPOSE ROOM", "CLASSROOM/LECTURE HALL/TRAINING ROOM - PENITENTIARY", "CLASSROOM/LECTURE HALL/TRAINING ROOM - ALL OTHER"]: reduction_factor_b = data_lookup(table_G3_7, lighting_space_type_p, "FULL-AUTO ON")`  
+
+          - Else, set baseline reduction factor to 0: `else: reduction_factor_b = 0`  
+
+        - For each lighting in space: `lighting_p in space_p.interior_lighting:`  
+
+          - Get lighting control type: `lighting_control_type_p = lighting_p.occupancy_control_type`  
+
+          - Get lighting occupancy sensor reduction factor from Table G3.7: `reduction_factor_p = data_lookup(table_G3_7, lighting_space_type_p, lighting_control_type_p)`  
+
```

Comment:
I think here we need to calculate what the correct proposed normalized schedule should be (based on its difference from the baseline) and compare it to the modeled proposed normalized lighting schedule for each space.

I would establish the correct schedule based on (a) schedule reduction fraction in Table G3.7 and (b) whether this fraction should be increased based on the listed controls for each interiorlighting object as required in footnotes b (b. For manual-ON or partial-auto-ON occupancy sensors, the occupancy sensor reduction factor shall be multiplied by 1.25.) and c (c. For occupancy sensors controlling individual workstation lighting, occupancy sensor reduction factor shall be 30%.)